hydrological-processes.csl

```diff
@@ -0,0 +1,220 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Hydrological Processes</title>
+    <id>http://www.zotero.org/styles/hydrological-processes</id>
+    <link href="http://www.zotero.org/styles/hydrological-processes" rel="self"/>
+    <link href="http://www.zotero.org/styles/aquatic-conservation" rel="template"/>
+    <link href="http://onlinelibrary.wiley.com/journal/10.1002/%28ISSN%291544-8452/homepage/ForAuthors.html" rel="documentation"/>
+    <author>
+      <name>Aditi Bhaskar</name>
+      <email>abhaskar@usgs.gov</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="geography"/>
+    <issn>0885-6087</issn>
+    <issn>1099-1085</issn>
+    <eissn>1099-1085</eissn>
```

Comment:
I see two ISSNs, one for print and one online.  So is the first one the
main ISSN and the second one for eISSN?

*Print ISSN:* 0885-6087
*Online ISSN:* 1099-1085

http://authorservices.wiley.com/bauthor/Aims_scope.asp?ref=0885-6087&site=1

On Tue, Aug 4, 2015 at 2:36 PM, Philipp Zumstein <notifications@github.com>
wrote:

> In hydrological-processes.csl
> <https://github.com/citation-style-language/styles/pull/1643#discussion_r36225126>
> :
>
> > +<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
> > +  <info>
> > +    <title>Hydrological Processes</title>
> > +    <id>http://www.zotero.org/styles/hydrological-processes</id>
> > +    <link href="http://www.zotero.org/styles/hydrological-processes" rel="self"/>
> > +    <link href="http://www.zotero.org/styles/aquatic-conservation" rel="template"/>
> > +    <link href="http://onlinelibrary.wiley.com/journal/10.1002/%28ISSN%291544-8452/homepage/ForAuthors.html" rel="documentation"/>
> > +    <author>
> > +      <name>Aditi Bhaskar</name>
> > +      <email>abhaskar@usgs.gov</email>
> > +    </author>
> > +    <category citation-format="author-date"/>
> > +    <category field="geography"/>
> > +    <issn>0885-6087</issn>
> > +    <issn>1099-1085</issn>
> > +    <eissn>1099-1085</eissn>
>
> Preferable, the eissn should stay, that seems to be correct.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/citation-style-language/styles/pull/1643/files#r36225126>
> .
>



-- 
Aditi S. Bhaskar
NSF Postdoctoral Research Fellow
U.S. Geological Survey, Eastern Geographic Science Center
12201 Sunrise Valley Drive, MSN 521, Reston, VA 20192
Phone: (703) 648-5157
http://aditibhaskar.weebly.com/